common/src/main/java/com/alibaba/nacos/common/executor/ExecutorFactory.java

```diff
@@ -27,70 +27,169 @@
 /**
  * Unified thread pool creation factory, and actively create thread
  * pool resources by ThreadPoolManager for unified life cycle management
+ * {@link ExecutorFactory.Managed}
+ *
+ * Unified thread pool creation factory without life cycle management
+ * {@link ExecutorFactory}
  *
  * @author <a href="mailto:liaochuntao@live.com">liaochuntao</a>
  */
 @SuppressWarnings("PMD.ThreadPoolCreationRule")
 public final class ExecutorFactory {
 
-    private static final ThreadPoolManager THREAD_POOL_MANAGER = ThreadPoolManager.getInstance();
+    public static ExecutorService newSingleExecutorService() {
+        ExecutorService executorService = Executors.newFixedThreadPool(1);
+        return executorService;
```

Comment:
return directly, do not use tmp variable.

like `return Executors.newFixedThreadPool(1);`

follow methods are the same.

Reply:
ok